<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DIalog</class>
 <widget class="QDialog" name="DIalog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>143</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Προσθήκη Προϊόντος</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="QLineEdit" name="nameInputBox">
     <property name="placeholderText">
      <string>Π.Χ.: Κομπόστα Ροδάκινο</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="unitInputBox">
     <property name="placeholderText">
      <string>Π.Χ.: Kg</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>'Ονομα προϊόντος:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLineEdit" name="amountInputBox">
     <property name="placeholderText">
      <string>Π.Χ.: 15</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="amountLabel">
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
       <underline>true</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>Ποσότητα Προϊόντος:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="unitInputLabel">
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Μονάδα Μέτρησης: </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nameInputBox</tabstop>
  <tabstop>amountInputBox</tabstop>
  <tabstop>unitInputBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DIalog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DIalog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
